developer_manual/client_apis/WebDAV/basic.rst

```diff
@@ -78,28 +100,184 @@ You can request additional properties by sending a request body with the :code:`
+|                               |                                                  | |      depth: ``0``                                                      |
+|                               |                                                  | |      owner: ``Jane Smith``                                             |
+|                               |                                                  | |      timeout: ``Infinite``                                             |
+|                               |                                                  | |      locktoken: { href: ``urn:uuid:f81de2ad-7f3d-a1b2-4f3c-`` }        |
+|                               |                                                  | |      lockroot: { href: ``http://www.example.com/container/`` }         |
+|                               |                                                  | | } ]                                                                    |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:supportedlock />          | | Provide the lock capabilities                  | [ { lockscope: ``exclusive``, locktype: ``write`` } ]                    |
+|                               | | supported by the resource.                     |                                                                          |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:id />                     | | The fileid namespaced by the instance id.      |                                                                          |
+|                               | | Globally unique.                               |                                                                          |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:fileid />                 | The unique id for the file within the instance.  |                                                                          |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:downloadURL />            | The URL to download the file.                    |                                                                          |
++-------------------------------+--------------------------------------------------+--------------------------------------------------------------------------+
+| <oc:permissions />            | The permissions that the user has over the file. | | S: Shared                                                              |
```

Comment:
Note down that the value is a string that contains all the letters for the given permissions, otherwise someone might thing it's an array or something else.